@eLBati , @kryskool : need advice :
1/ why travis highlight this error, it is a non sens:
./account_invoice_sale_origin/**init**.py:22:1: F401 'sale' imported but unused

2/ There is errors on other modules YAML files; what should I do with that

It starts to be discouraging to publish....

--- a/account_invoice_sale_origin/__init__.py
+++ b/account_invoice_sale_origin/__init__.py
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-##############################################################################
+###############################################################################
 #
 #    OpenERP, Open Source Management Solution
 #    Copyright (c) 2010-2013 Savoir-faire Linux. All Rights Reserved.
@@ -17,6 +17,6 @@
 #    You should have received a copy of the GNU Affero General Public License
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
-##############################################################################
+###############################################################################
 
-from .import sale
+from . import sale

--- a/account_invoice_sale_origin/__openerp__.py
+++ b/account_invoice_sale_origin/__openerp__.py
@@ -24,7 +24,7 @@
     'version': '1.1',
     'category': 'Finance',
     'description': """
-    
+
     This module displays the origin field of account invoice lines and populate
     it when a customer invoice is create from a sale order.
 

